litmus-portal/graphql-server/pkg/database/mongodb/chaoshub/operations.go

```diff
@@ -9,9 +9,25 @@ import (
 	"go.mongodb.org/mongo-driver/bson"
 )
 
+type Operator interface {
+	CreateChaosHub(ctx context.Context, chaosHub *ChaosHub) error
+	GetChaosHubByProjectID(ctx context.Context, projectID string) ([]ChaosHub, error)
+	GetHubs(ctx context.Context) ([]ChaosHub, error)
+	GetHubByID(ctx context.Context, hubID string, projectID string) (ChaosHub, error)
+	UpdateChaosHub(ctx context.Context, query bson.D, update bson.D) error
+}
+
+type chaosHubOperator struct {
+	operator mongodb.MongoOperator
+}
+
+func NewOperator() Operator {
+	return &chaosHubOperator{operator: &mongodb.MongoOperations{}}
+}
```

Comment:
We probably don't need to convert this to an interface because `mongodb.Operator` which was previously used in the functions is already an interface type (`MongoOperator`). If this is for mocking in tests imo just creating an in memory db mock that implements `mongodb.MongoOperator` would suffice instead of having to mock every single db wrapper that we have for each service. That will be much easier and faster imo. 

Though I agree that we should not be using a global shared variable `mongodb.Operator` so the best change would be to just create a strict type for each service (in this case ChaosHubOperator) and in the initializer, we take the db implementation needed something like this :

```
func NewChaosHubOperator(dbOperator mongodb.MongoOperator) ChaosHubOperator {
	return &ChaosHubOperator{operator: dbOperator}
}
```
That way we can substitute the db instance to mock db in our tests without having to write mocks for each individual service wrapper. WDYT?